src/controllers/post.controller.js

```diff
@@ -1,13 +1,29 @@
 import urlMetadata from "url-metadata";
 import { insertNewPost, getPosts } from "../repositories/post.repository.js";
+import { getTag, insertNewTag, insertNewTagQuote } from "../repositories/hashtags.repositories.js";
 
 async function createPost(req, res) {
   const { url, description } = req.body;
   const userId = res.locals.userId;
 
+  const tags = description.split(' ').filter((word) => word[0]==='#' && word.length > 1).map((word) => word.slice(1));
```

Comment:
**suggestion:** I believe that this logic could be implemented in an isolated function that would serve as a helper

Reply:
Saulo, I considered it and I just though that it can be re-used at the edit post feature. If I isolate it as a function, in which file do you suggest I should declare and export it?

Reply:
I would create a new folder called utils in src, for the file I would think of something like getTags, WDYT?

Reply:
Ok, nice! I will refactor it and update here. Thanks, Saulo